test/integration/fixtures/VerticalLinearStepper.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {
+  Step,
+  Stepper,
+  StepLabel,
+  StepContent,
+} from 'src/Stepper';
+import RaisedButton from 'src/RaisedButton';
+import FlatButton from 'src/FlatButton';
+
+class VerticalLinearStepper extends React.Component {
+
+  static contextTypes = {muiTheme: React.PropTypes.object.isRequired};
+
+  state = {
+    finished: false,
+    stepIndex: 0,
+  };
+
+  handleNext = () => {
+    const {stepIndex} = this.state;
+    this.setState({stepIndex: stepIndex + 1});
+  };
+
+  handlePrev = () => {
+    const {stepIndex} = this.state;
+    this.setState({stepIndex: stepIndex - 1});
+  };
+
+  renderStepActions(step) {
+    const {stepIndex} = this.state;
+
+    return (
+      <div style={{margin: '12px 0'}}>
+        <RaisedButton
+          label={stepIndex === 2 ? 'Finish' : 'Next'}
+          disableTouchRipple={true}
+          disableFocusRipple={true}
+          primary={true}
+          onTouchTap={this.handleNext}
+          style={{marginRight: 12}}
+        />
+        {step > 0 && (
+          <FlatButton
+            label="Back"
+            disabled={stepIndex === 0}
+            disableTouchRipple={true}
+            disableFocusRipple={true}
+            onTouchTap={this.handlePrev}
+          />
+        )}
+      </div>
+    );
+  }
+
+  render() {
+    const {stepIndex} = this.state;
+
+    return (
+      <Stepper activeStep={stepIndex} orientation="vertical" style={{width: 380, margin: 'auto'}}>
+        <Step>
+          <StepLabel>Select campaign settings</StepLabel>
+          <StepContent>
+            <p>
+              For each ad campaign that you create, you can control how much
+              you're willing to spend on clicks and conversions, which networks
```

Comment:
I've seen this in a few places when reviewing diffs/source code lately... really ugly :(